[4.1.0] Add Restrict Unassigned Scope Info to the Doc (#8090)
wso2/docs-apim: en/docs/design/api-security/oauth2/oauth2-scopes/fine-grained-access-control-with-oauth-scopes.md

     If you first create a local scope and then create a shared scope with same scope name before attaching the local scope to any API resource, the local scope will removed from local scope UI. However, it will not be reflected in the API Definition, unless you save the API. This is an identified limitation in supporting both local and shared scopes.

## Restrict Unassigned Scopes

The default behavior of the Scope Issuer in API-M is not to restrict any unregistered/unassigned scopes when obtaining a token. For example, if you create a local scope for an API but do not assign it to any API resource, obtaining a token using the local scope name will include that scope in the returned token.

To override this behavior and validate unregistered/unassigned scopes when obtaining a token, set the Java system property `restrict.unassigned.scopes` to `true` at the server startup:
This can be done in one of two ways.

> **Option 1**: Adding in startup script

Reviewer: ```suggestion
**Option 1**: Adding it into the startup script
```

- Open `<API-M_HOME>/bin/api-manager.(sh|bat)` file.

> - Add `-Drestrict.unassigned.scopes=true` at the end of the file.

Reviewer: Let's provide an example here as we need to add the line separator `\`. Take 2 surrounding lines and give the example.

- Restart the server.

>   **Option 2**: Provide as a parameter during server startup

Reviewer: ```suggestion
  **Option 2**: Provide as a system property during server startup
```

Author: Resolved with with the latest commit

>    Restart the server with the parameter set as below.

Reviewer: ```suggestion
   Restart the server with the property set as below.
```

Author: Resolved with with the latest commit
